### ISSUE TITLE
scroll issue with keyboard arrows

### ISSUE THREAD

**Issue by mecs13** [mecs13/website_text#12]
+- Click "who we are" 
+- see intro on right column  
+- cannot scroll through with arrows
+
kyle os and hardware

**Comment by tbinetruy** [comment]
this is a hard bug to solve. It's because I have to override the scroll to get the parallax. So don't be surprised if you see this hang on for a little bit while I get the other bugs out of the way.
